tests/unit/zen_server/test_zen_server.py

```diff
@@ -32,7 +32,9 @@ def running_zen_server(tmp_path_factory: pytest.TempPathFactory) -> ZenServer:
     tmp_path = tmp_path_factory.mktemp("local_zen_store")
     global_cfg = GlobalConfiguration()
 
-    backing_zen_store = LocalZenStore().initialize(str(tmp_path))
+    backing_zen_store = SqlZenStore().initialize(
+        f"sqlite:///{tmp_path / 'store.db'}"
```

Comment:
Is this something that needs backslashes on windows instead or fine with the hardcoded `/`?

Reply:
I guess it should be fine since all the Windows unit tests passed in the CI?

Reply:
I think we actually skip all the ZenServer tests on windows as it doesn't support the daemon process?

Reply:
So then it doesn't matter anyways? 😄